IBM Cloud/WML/notebooks/Misc/Configure subscription monitors using training statistics.ipynb

```diff
@@ -28,13 +27,12 @@
    "outputs": [],
    "source": [
     "!pip install pyspark==3.0.3 --no-cache | tail -n 1\n",
```

Comment:
We should install pyspark 3.2.x because that's what is given below spark-mllib_3.2